cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress"]
```

Comment:
@florkbr I'm not seeing any common options between this config and the files under https://github.com/openshift/dynamic-plugin-sdk/tree/main/packages/common/tsconfig-bases. The only common field is "include" but the path would be different.

Reply:
We could use `app-react-esm.json` as the base config here, it's already used by both sample plugin and sample app packages.

This ^^ config basically represents a TS web project targeting reasonably recent web browsers (`es2020`).

I don't think we need an explicit `lib` setting (`dom` should be included by default, but I might be wrong).

Also I think we can skip the `types` setting since the default is very reasonable

> By default all visible `”@types”` packages are included in your compilation.

```json
{
  "extends": "@monorepo/common/tsconfig-bases/app-react-esm.json",
  "compilerOptions": {
    "outDir": "dist"
  },
  "include": ["**/*.ts"],
  "exclude": ["node_modules"]
}
```